assignments/closure.js

```diff
@@ -1,20 +1,48 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function myClosure() {
+  const  name = "Talent"; // name is a local variable created by closure declare as a function
+  function displayName() { // displayName() is the inner function, a closure
+      console.log(name); // displayName() uses variable declared in the parent function    
+  }
+  displayName();    
+}
 
 // ==== Challenge 2: Create a counter function ====
-const counter = () => {
   // Return a function that when invoked increments and returns a counter variable.
-};
 // Example usage: const newCounter = counter();
-// newCounter(); // 1
-// newCounter(); // 2
+const counter = () => {
+ let  counter  = 0;
+          return function newCounter(){
+            ++counter;
```

Comment:
You forgot to add `return` inside the function. Without it won't increment properly because the function is not returning anything.